dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -1137,4 +1136,45 @@ public ResponseEntityPageWorkflowActionsView findAvailableActions(@Context final
 
         throw new DoesNotExistException("The page: " + findAvailableActionsForm.getPath() + " do not exist");
     } // findAvailableActions.
+
+    /**
+     * Receives the Identifier of an HTML Page, returns all the available languages in dotCMS and,
+     * for each of them, adds a flag indicating whether the page is available in that language or
+     * not. This may be particularly useful when requiring the system to provide a specific action
+     * when a page is NOT available in a given language. Here's an example of how you can use it:
+     * <pre>
+     *     GET <a href="http://localhost:8080/api/v1/page/${PAGE_ID}/languages">http://localhost:8080/api/v1/page/${PAGE_ID}/languages</a>
+     * </pre>
+     *
+     * @param request  The current instance of the {@link HttpServletRequest}.
+     * @param response The current instance of the {@link HttpServletResponse}.
+     * @param pageId   The Identifier of the HTML Page whose available languages will be checked.
+     *
+     * @return A {@link Response} object containing the list of languages and the flag indicating
+     * whether the page is available in such a language or not.
+     */
+    @GET
+    @Path("/{pageId}/languages")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public Response checkPageLanguageVersions(@Context final HttpServletRequest request,
+                                              @Context final HttpServletResponse response,
+                                              @PathParam("pageId") final String pageId) {
+        final User user = new WebResource.InitBuilder(webResource).requestAndResponse(request, response)
+                .rejectWhenNoUser(true)
+                .requiredBackendUser(true)
+                .init().getUser();
+        Logger.debug(this, String.format("Check the languages that page '%s' is available on", pageId));
+        try {
+            final List<ExistingLanguagesForPageView> languagesForPage =
+                    this.pageResourceHelper.getExistingLanguagesForPage(pageId, user);
+            return Response.ok(new ResponseEntityView<>(languagesForPage)).build();
+        } catch (final Exception e) {
```

Comment:
do not need to catch, jersey took and map the exception

Reply:
Done!